bin/check_libs

```diff
@@ -48,7 +48,7 @@ while($dir = shift) {
 
   print "finding ELF objects...\n";
 
-  my $cmd = "cd $dir; find . -type f -o -type l | grep -v modules | xargs file -L";
+  my $cmd = "cd $dir; find . -type f | grep -v modules | xargs file -L";
```

Comment:
NP: Since this will no longer find symlinks, the `-L` of the `file` command is now no longer needed, I guess.

Reply:
Might be; but it seems the symlinks are really needed - I'll recheck.